src/components/__tests__/search-bar-test.jsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-env jest */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import SearchBar from '../search-bar/search-bar';
+
+describe('search-bar', () => {
+  it('Renders the search-bar', () => {
+    const sBar = shallow(<SearchBar />);
```

Comment:
This is a minor nit, but would you please avoid abbreviations like this? It’s often surprising what someone *else* might read from an abbreviation you thought was obvious. https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md#naming